packages/components/src/components/checkbox/checkbox.stories.ts

```diff
@@ -0,0 +1,289 @@
+import '../../solid-components';
```

Comment:
Could be nice to keep sizes on y axis and add descriptions for all stories to be consistent.  
Just my 2 cents not a requirement.